psydac/feec/multipatch/plotting_utilities.py

```diff
@@ -248,6 +308,8 @@ def my_small_plot(
         gridlines_x2=None,
         surface_plot=False,
         cmap='viridis',
+        cb_min=None,
+        cb_max=None,
         save_fig=None,
         save_vals = False,
         hide_plot=False,
```

Comment:
Could you add a docstring to this function? The number of arguments has become large...